GO.pm

```diff
@@ -30,10 +30,35 @@ limitations under the License.
  mv GO.pm ~/.vep/Plugins
  ./vep -i variations.vcf --plugin GO
 
+ # input a custom directory where to write and read GFF files with GO terms
+ ./vep -i variations.vcf --plugin GO,${HOME}/go_terms
+ 
+ # use remote connection (available for compatibility purposes)
+ ./vep -i variations.vcf --plugin GO,remote
+
 =head1 DESCRIPTION
 
- A VEP plugin that retrieves Gene Ontology terms associated with
- transcripts/translations via the Ensembl API. Requires database connection.
+ A VEP plugin that retrieves Gene Ontology (GO) terms associated with
+ transcripts (e.g. GRCh38) or their translations (e.g. GRCh37) from a custom GFF
+ file. This GFF file is automatically created (if the input file does not exist)
+ by querying the Ensembl core database, according to database version, species
+ and assembly used in VEP.
+ 
+ The GFF file containing the GO terms is saved to and loaded from the working
+ directory by default. To change this, provide a directory path as an argument:
+ 
+   --plugin GO,${HOME}/go_terms
+ 
+ The GNU zgrep and GNU sort commands must be installed in your path to create
+ the custom GFF file. The tabix and bgzip utilities are also required: check
+ https://github.com/samtools/htslib.git for installation instructions.
+ 
+ Alternatively, for compatibility purposes, the plugin allows to use a remote
+ connection to the Ensembl API by using "remote" as a parameter. This remote
+ connection retrieves GO terms one by one at both the transcript and translation
+ level:
+
+   --plugin GO,remote
  
```

Comment:
"The tabix utility must be installed in your path to use this plugin in an offline mode" 
 Suggesting that this is added to the documentation

Reply:
Edited the plugin description for clarity, please see if it is better this way.